tensorflow/c/experimental/filesystem/modular_filesystem.h

```diff
@@ -61,34 +61,34 @@ class ModularFileSystem final : public FileSystem {
 
   Status NewRandomAccessFile(
       const std::string& fname,
-      std::unique_ptr<RandomAccessFile>* result) override;
+      std::unique_ptr<RandomAccessFile>* result, TransactionToken* token=nullptr) /*override*/;
```

Comment:
Why the removal of `override`?

Reply:
Oh, because base class is not changed yet and we cannot change it before these change. Makes sense